### PR TITLE
Add Group Entity PR Card

### PR DESCRIPTION
**Pull request by dlaird-ovo**
This adds a card that can be used on Group Entity pages , if the Group has the:

```
github.com/team-slug: <team_name>
```

All reviews assigned to the ```<team_name>``` will then be shown - helping identity pending reviews.

Card looks like this (redacted repo names)
Team_Pull_Requests

#### :heavy_check_mark: Checklist

- [X] Added tests for new functionality and regression tests for bug fixes
- [X] Added changeset (run `yarn changeset` in the root)
- [X] Screenshots of before and after attached (for UI changes)
- [X] Added or updated documentation (if applicable)

### CODE REVIEW

**Comment by dlaird-ovo** [comment]
Sorry, will resolve the style/prettier issues later today and get it updated

File: .changeset/slow-zebras-clap.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@roadiehq/backstage-plugin-github-pull-requests': major
```
Comment (kissmikijr) [inline .changeset/slow-zebras-clap.md:2]:
```suggestion
'@roadiehq/backstage-plugin-github-pull-requests': minor
```
Minor version bump should be enough as we do not break the API.

**Review by kissmikijr — CHANGES_REQUESTED** [review]
Nice work, thank you! :) Just a tiny request for the version bump!

File: .changeset/slow-zebras-clap.md (hunk 1 above)
Comment (dlaird-ovo) [reply .changeset/slow-zebras-clap.md]: Done 👍

**Pull request merged**